templates/terragrunt_bootstrap/content/terragrunt/aws/README.md

```diff
@@ -0,0 +1,5 @@
+# In the aws directory, we place the terraform files to create the infrastructure in AWS. General format of the directory structure is as follows:
+# Cloud computing service directory such as Cloudfront, S3, ECS, RDS, etc. Inside the directory, we can have the following files:
+    # |__ Input.tf file to define the input variables
+    # |__ Main.tf file to define the resources
+    # |__ Output.tf file to define the output variables
```

Comment:
Minor nit - could you make these all lowercase just to match our naming convention in other projects:
```suggestion
    # |__ input.tf file to define the input variables
    # |__ main.tf file to define the resources
    # |__ output.tf file to define the output variables
```

Reply:
Good point Pat - I have corrected this.